src/lib/client.js

```diff
@@ -1,6 +1,6 @@
 import CozyClient from 'cozy-client'
 
-const client = new CozyClient({
+const client = CozyClient.fromDOM({
```

Comment:
Aaaah, makes sense now! So previously we had an instance of CozyClient without having it 'logged' or else 🤔

Reply:
I don't understand the " without having it 'logged' or else 🤔" part ?